response/core/serializers.py

```diff
@@ -18,6 +19,11 @@ class Meta:
         fields = ("id", "timestamp", "text", "event_type", "metadata")
         read_only_fields = ("id",)
 
+    def to_representation(self, instance):
+        rep = super().to_representation(instance)
+        rep["text"] = slack_to_human_readable(rep["text"])
```

Comment:
Does `slack_to_human_readable` use the users in the DB, or the slack client to resolve these?  If the latter, we should change it to use the DB, or else the API will be making loads of synchronous API calls to resolve all the references.

Reply:
I've changed this to use the cached function 👍 

Also moved the reference util functions out into a separate module for cleaner separation.